testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -294,28 +296,24 @@ impl BitcoinRegtestController {
             db: None,
             burnchain_db: None,
             chain_tip: None,
-            burnchain_config: None,
+            burnchain_config: burnchain,
             ongoing_block_commit: None,
             should_keep_running: None,
             allow_rbf: true,
         }
     }
 
-    fn default_burnchain(&self) -> Burnchain {
-        let (network_name, _network_type) = self.config.burnchain.get_bitcoin_network();
-        match &self.burnchain_config {
-            Some(burnchain) => burnchain.clone(),
-            None => {
-                let working_dir = self.config.get_burn_db_path();
-                match Burnchain::new(&working_dir, &self.config.burnchain.chain, &network_name) {
-                    Ok(burnchain) => burnchain,
-                    Err(e) => {
-                        error!("Failed to instantiate burnchain: {}", e);
-                        panic!()
-                    }
-                }
-            }
-        }
+    fn default_burnchain(config: &Config) -> Burnchain {
+        let (network_name, _network_type) = config.burnchain.get_bitcoin_network();
+        let working_dir = config.get_burn_db_path();
+        let mut burnchain = Burnchain::new(&working_dir, &config.burnchain.chain, &network_name)
+            .unwrap_or_else(|e| {
```

Comment:
This can be replaced with `.expect("Failed to instantiate burnchain")`, which will implicitly log the error.

Reply:
Done!